zebra-network/src/meta_addr.rs

```diff
@@ -640,6 +661,7 @@ impl MetaAddrChange {
             //   reconnection attempts.
             UpdateResponded { .. } => Some(DateTime32::now()),
             UpdateFailed { .. } => None,
+            AddPeer { .. } => None,
         }
     }
 
```

Comment:
There's a typo in the comment just below this line, would you mind fixing it as part of this PR?

It should say "last failure" to match the method name.